### DISCUSSION TITLE
outgoing ping not catch by OpenSnitch 1.5.2

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#897] [Q&A] [answered]
Hi everyone,

I would like to know if it's normal that the version 1.5.2 do not catch outgoing ping (echo-request)

I'm using nftables (just in case)

Cheers

**Comment by BobSquarePants** [comment, marked as answer]
Sorry it's a duplicate of : https://github.com/evilsocket/opensnitch/issues/714

**Comment by gustavo-iniguez-goya** [comment]
Yes @BobSquarePants , it's normal. The change was only added to master branch, so v1.6.x will include it.
